security/pkg/nodeagent/sds/server.go

```diff
@@ -193,15 +202,23 @@ func (s *Server) initGatewaySdsService(options *Options) error {
 	}
 
 	go func() {
+		log.Info("Start SDS grpc server for ingress gateway proxy")
```

Comment:
Please change to expotential back off, which is really simple. Psedu code.

```go
waitTime := time.Second
for i := 0; i < 10; i++ {
  log.Errorf("Failed to start gRPC server for SDS, retry in %v", waitTime)
  time.Sleep(waitTime)
  waitTime *= 2
}
```

Reply:
Done: the PR has been revised accordingly.